jekyll/_cci2/schedule-pipelines-with-multiple-workflows.adoc

```diff
@@ -17,22 +17,36 @@ contentTags:
 [#introduction]
 == Introduction
 
-As a scheduled pipeline is a triggered pipeline, and it will run every workflow in the configuration. You can set conditionals to keep the scheduled pipelines running only when you want them to run. One way is through built-in pipeline values, and one way is through custom pipeline parameters.
+Scheduled pipelines are simply triggered pipelines, and by default, for each trigger, every workflow that has been configured will run. You can filter workflows in your `.circleci/config.yml` file with conditionals. The conditionals you set will control which workflows will run when a scheduled pipeline is triggered. These conditions can be set with built-in pipeline values, or through custom pipeline parameters. The examples below demonstrate various ways to set conditionals in your `.circleci/config.yml` file.
 
-[#filter-using-built-in-pipeline-values]
-## Filter using built-in pipeline values
+[#schedule-using-built-in-pipeline-values]
+## Schedule using built-in pipeline values
 
-One way to implement workflows filtering is by using pipeline values. The example below uses the built in pipeline values `pipeline.trigger_source` and `pipeline.schedule.name`.
+One way to implement workflows filtering is by using pipeline values. The example below uses the built-in pipeline values `pipeline.trigger_source` and `pipeline.schedule.name`. The examples use `daily_build` and `nightly_build` as the `pipeline.schedule.name`, however, the value of `pipeline.schedule.name` can be whatever you would like when using scheduled pipelines.
```

Comment:
```suggestion
One way to implement workflows filtering is by using pipeline values. The example below uses the built-in pipeline values `pipeline.trigger_source` and `pipeline.schedule.name`. The examples use `daily_build` and `nightly_build` as the `pipeline.schedule.name`, however, the value of `pipeline.schedule.name` can be whatever you would like.
```

Reply:
@rosieyohannan I'm going to keep this as is, because when you _aren't_ using scheduled pipelines, there is a default naming convention that you cannot override for `pipeline.schedule.name`.